src/hotspot/os/posix/os_posix.cpp

```diff
@@ -854,6 +854,14 @@ void os::_exit(int num) {
   ALLOW_C_FUNCTION(::_exit, ::_exit(num);)
 }
 
+bool os::dont_yield() {
+  return DontYieldALot;
+}
```

Comment:
This could have been placed in os.hpp/cpp and removed from all os specific files, but we will be deleting it in JDK 24 anyway.